### DISCUSSION TITLE
报错BOM字符了怎么办啊

### DISCUSSION THREAD

**Discussion by HasueYuki** [azmiao/uma_plugin#41] [Q&A]
![QQ截图20220905164134](https://user-images.githubusercontent.com/39873270/188407257-c867d39e-f53a-4a5f-b0cc-eecb3de2e2b0.png)
像这样怎么办啊

**Comment by azmiao** [comment]
看看你的properties.json

**Reply by HasueYuki** [reply]
![QQ截图20220905185835](https://user-images.githubusercontent.com/39873270/188433727-dc3a1e56-5efa-49c3-972e-195f3603c5b9.png)

**Comment by azmiao** [comment]
话说你有没有用txt修改过文件，txt容易出问题，如果修改过，请先删除该配置文件，然后使用命令git pull -f恢复默认状态

**Reply by HasueYuki** [reply]
也没有啊，而且这问题是今天突然出现的
昨天都还正常的，我都没动过，今天突然就寄了

**Reply by azmiao** [reply]
那就奇怪了，我这边自己用的没啥问题，要不还是先删除该配置文件，然后使用命令git pull -f恢复一下默认状态吧，我这几天再试试能不能复现问题

**Reply by HasueYuki** [reply]
好吧我试试

**Reply by HasueYuki** [reply]
我重装了整个插件之后就好了，之前的问题是所有涉及图片的都会报BOM字符的错

**Reply by azmiao** [reply]
> 我重装了整个插件之后就好了，之前的问题是所有涉及图片的都会报BOM字符的错

嗯嗯，这种情况一般是配置文件内含有特殊字符导致读取失败，最常见的情况就是windows记事本，反正现在能用就行，明天如果还有问题可以再叫我一下

**Reply by HasueYuki** [reply]
好的谢谢
